content/en/docs/setup/cri.md

```diff
@@ -0,0 +1,169 @@
+---
+reviewers:
+- vincepri
+- bart0sh
+title: CRI installation
+content_template: templates/concept
+weight: 100
+---
+{{% capture overview %}}
+Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+This page contains installation instruction for various runtimes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+### Docker
+
+On each of your machines, install Docker.
+Version 17.03 is recommended, but 1.11, 1.12 and 1.13 are known to work as well.
```

Comment:
i think we should recommending 18.06 here as well.
`Version 18.06 is recommended, but 1.11, 1.12, 1.13 and 17.03` are known to work as well.

Reply:
[1]

Reply:
@neolit123 Thanks for pointing this out. Should be fixed now.